e2e/tests/home/home-page.spec.ts

```diff
@@ -1,63 +0,0 @@
-import BaseElement from 'app/element/base-element';
-import HomePage from 'app/page/home-page';
-import WorkspaceCard from 'app/component/card/workspace-card';
-import { signInWithAccessToken } from 'utils/test-utils';
-import { getStyleValue } from 'utils/element-utils';
-
-describe('Home page ui tests', () => {
-  beforeEach(async () => {
-    await signInWithAccessToken(page);
-  });
-
-  test('Check visibility of Workspace cards', async () => {
-    await checkCreateNewWorkspaceLink();
-
-    const allCards = await new WorkspaceCard(page).findAllCards();
```

Comment:
Interestingly, this test relies on workspaces created by other tests - it doesn't create any itself. If there are no workspaces (or worse, there are workspaces but the cards are failing to appear) the test still passes.